src/PrestaShopBundle/Controller/Admin/Configure/AdvancedParameters/WebserviceController.php

```diff
@@ -59,8 +59,23 @@ class WebserviceController extends FrameworkBundleAdminController
     public function indexAction(WebserviceKeyFilters $filters, Request $request)
     {
         $form = $this->getFormHandler()->getForm();
+        $gridWebserviceFactory = $this->get('prestashop.core.grid.factory.webservice_key');
```

Comment:
One think I don't get here is that this is the exact same code as in the `renderPage` So what's the point of duplicating this code instead of using the existing method?